packages/plugin-ext/src/main/browser/webview/webview.ts

```diff
@@ -450,8 +450,12 @@ export class WebviewWidget extends BaseWidget implements StatefulWidget, Extract
             }
             return link;
         }
-        if (!!this.contentOptions.enableCommandUris && link.scheme === Schemes.command) {
-            return link;
+        if (link.scheme === Schemes.command) {
+            if (Array.isArray(this.contentOptions.enableCommandUris) && this.contentOptions.enableCommandUris.some(value => value === link.path.toString())) {
+                return link;
+            } else if (this.contentOptions.enableCommandUris === true) {
+                return link;
+            }
```

Comment:
Suggestion: Just because it seems weird to me to have two branches returning the same value.
```suggestion
            if (this.contentOptions.enableCommandUris === true || Array.isArray(this.contentOptions.enableCommandUris) && this.contentOptions.enableCommandUris.some(value => value === link.path.toString())) {
                return link;
            }
```

Reply:
It breaks the max line length ESLint rule, and even when I tried adding some line returns to make it look nice it didn't really look any nicer than what is written in this PR...

Let's leave it as is :)